app/controllers/admin/tenants_controller.rb

```diff
@@ -34,8 +34,8 @@ def update
     def destroy
       begin
         Apartment::Tenant.drop(@tenant.name)
-      rescue StandardError => e
-        logger.warn(e)
+      rescue StandardError => error
```

Comment:
__[RuboCop]__ Naming/RescuedExceptionsVariableName: Use `e` instead of `error`. ([link](https://sider.review/gh/repos/47168038/pulls/549#issue-4014094))

<sub>posted by Sider</sub>